lib/run-status.js

```diff
@@ -9,6 +9,7 @@ export default class RunStatus extends Emittery {
 		super();
 
 		this.pendingTests = new Map();
+		this.pendingTestsLogReference = new Map();
```

Comment:
separate map from `this.pendingTests` because they are modified and initialized in different events with different objects.

Reply:
Why `Reference`?

Reply:
oops, I think `pendingTestsLogs` sounds better